src/olympia/amo/management/commands/initialize.py

```diff
@@ -58,4 +58,4 @@ def handle(self, *args, **options):
         # We should reindex even if no data is loaded/modified
         # because we might have a fresh instance of elasticsearch
         else:
-            call_command('reindex', '--wipe', '--force', '--noinput')
```

Comment:
I wonder if we should keep the `--force` here. It prevents the error from being raise when there is already a reindexation in progress, which I think is what we want here, in case somehow it was triggered from elsewhere?

Reply:
Done. Also moved the check, above the check for already indexing.